### PR TITLE
AMP-81914 Add supported browser version

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

for ticket: https://amplitude.atlassian.net/browse/AMP-81914
image

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/typescript-browser/index.md [hunk 1]
```diff
@@ -294,7 +296,7 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 
 --8<-- "includes/sdk-ts-browser/shared-plugins.md"
 
-##### Web attribution enrichment plugin
+#### Web attribution enrichment plugin
```
Comment (yuhao900914) [inline docs/data/sdks/typescript-browser/index.md:299]: I feel we shouldn't change this. 
This the structure of shared-plugins:
#### Plugin examples
##### Destination type plugin
##### Enrichment type plugin
This should either be:
##### Web attribution enrichment plugin
or ###### Web attribution enrichment plugin

File: includes/sdk-browser-supported-version.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+!!! warning "Supported browser versions"
+    This SDK uses [nullish coalescing](https://caniuse.com/mdn-javascript_operators_nullish_coalescing) (`??`) and [optional chaining](https://caniuse.com/mdn-javascript_operators_optional_chaining) (`?.`) operators. Please refer to the links to see supported browser versions.
```
Comment (yuhao900914) [inline includes/sdk-browser-supported-version.md:2]: Should we make this more general?
`This SDK utilizes modern JavaScript features. Please refer to the provided links for browser compatibility information.
  - nullish coalescing
  - optional chaining
`

Comment (justin-fiedler) [reply includes/sdk-browser-supported-version.md:2]: I like this.

**Review by justin-fiedler — APPROVED** [review]

File: docs/data/sdks/typescript-browser/index.md (hunk 1 above)
Comment (Mercy811) [reply docs/data/sdks/typescript-browser/index.md:299]: Looks like there is no `#### Plugin examples`. But the structure would be

### Plugins
   #### Web attribution enrichment plugin
       ##### Differences from base SDK
   #### Page view enrichment plugin
       ##### Differences from base SDK

Thanks for calling out. Deleted a # for the later two

Comment (Mercy811) [reply docs/data/sdks/typescript-browser/index.md:299]: I was wrong. Plugin examples is embedded at a shared file.

Comment (Mercy811) [reply docs/data/sdks/typescript-browser/index.md]: Reverted it

**Pull request merged**